tests/integration/api/test_observation_table.py

```diff
@@ -56,6 +56,8 @@ async def test_observation_table_from_source_table(
     check_location_valid(table_details, session)
     await check_materialized_table_accessible(table_details, session, source_type, sample_rows)
 
+    assert observation_table.min_interval_secs_between_entities == 0
```

Comment:
this currently returns 0 because it seems like the observation table might have rows for an entity that have the same timestamp. should we filter the `POINT_IN_TIME` column for unique timestamps?

Reply:
I think it's better to test using an observation that returns non-zero for this new stats. Maybe we can create a dedicated observation table for it?